lib/firebase_options.dart

```diff
@@ -0,0 +1,70 @@
+// File generated by FlutterFire CLI.
+// ignore_for_file: lines_longer_than_80_chars, avoid_classes_with_only_static_members
+import 'package:firebase_core/firebase_core.dart' show FirebaseOptions;
```

Comment:
seems that we don't need this file

Reply:
why? we use it in presentation

Reply:
we can initialize Firebase without options. try to remove it from Firebase init and if everything works delete the file